numpy/linalg/lapack_lite/f2c_config.c

```diff
@@ -107,7 +106,6 @@ doublereal dlamch_(char *cmach)
 
 
     if (first) {
-	first = FALSE_;
```

Comment:
Because the oracle that is `f2c` decided to. This is generated code beyond my control